app/templates/views/pricing/how-to-pay.html

```diff
@@ -39,15 +39,23 @@ <h2 class="heading-medium" id="purchase-orders">
     </h2>
 
     <p>
-      If your organisation’s annual usage will cost over £500 (before <abbr title="Value Added Tax">VAT</abbr>) you need to provide us with a purchase order so we can invoice you.
+      If your organisation's estimated spend is more than £500 a year (before <abbr title="Value Added Tax">VAT</abbr>), you need to send us a purchase order (PO).
```

Comment:
```suggestion
      If your organisation’s estimated spend is more than £500 per year (before <abbr title="Value Added Tax">VAT</abbr>), you need to send us a purchase order (PO).
```
as per https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#money